.github/workflows/cmake.yml

```diff
@@ -24,7 +29,7 @@ jobs:
     - name: make build dir
       run: cmake -E make_directory ${{runner.workspace}}/build
     - name: configure cmake
-      run: cmake -S ${{ github.workspace }} -B ${{runner.workspace}}/build
+      run: cmake -S ${{ github.workspace }} -B ${{runner.workspace}}/build -DUSE_OPENSSL=${{ matrix.openssl }}
```

Comment:
do you think it'd be useful to test it with both options? Seems like we should test it both ways?

Reply:
it does use both options. see job.matrix